guides/store-retrieve-messages.md

```diff
@@ -0,0 +1,162 @@
+# Retrieve Messages Using Waku Store
+
+When running a web dApp or a mobile phone application,
```

Comment:
It is less about it being unreliable and more about phones and browsers being mostly-offline. It is expected behavior if you close your browser/background your app.

Reply:
Indeed, thanks!